projects/plugins/migration/src/class-wpcom-migration.php

```diff
@@ -61,7 +62,7 @@ function () {
 			},
 			1
 		);
-		\Jetpack_Backup::initialize();
+		My_Jetpack_Backup::initialize();
```

Comment:
I think it would be okay to keep it as `Jetpack_Backup` to avoid any confusions with `My_Jetpack`, which is another feature in our plugins.

Reply:
Good point, renamed back in 3e8934a.